styleguide/components/41-organisms/feedback-form/feedback-form.twig

```diff
@@ -0,0 +1,39 @@
+<div class="feedback-form {{ modifier }}">
+  <h3><button>{{ 'Can we improve something about this page?' }}<i class="icon-chevron-down" aria-hidden="true"></i></button></h3>
+  <div class="content" aria-hidden="true">
```

Comment:
This needs an ID, you'll want to reference it from the button as 'aria-controls'.

Reply:
OK.